### PR TITLE
small improvements to documentation

### PR DESCRIPTION
**Pull request by cvasilak**
some improvements:
 - added heroku screenshots
 - added links to pelion documentation
 - an animated gif of the values changing instead of the screenshot (https://i.ibb.co/8bwXbLS/web-app-demo.gif)

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -11,20 +11,18 @@ This app consists of
...
 
 ## Visualising Data
 
-The included react app requests all data from the server and visualises this using the recharts library
+The included react app requests all data from the server and visualises this using the [Recharts](http://recharts.org/en-US/) library.
 
-![](docs/view_data.png)
-
-![](docs/app.png)
+![](https://i.ibb.co/8bwXbLS/web-app-demo.gif)
```
Comment (froeba) [inline ARCHITECTURE.md]: This gif doesn't seem to be animated.  What is it replacing?

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -45,6 +48,9 @@ This app requires access to a Postgres database to store resource values. Heroku
 1. Navigate to `Resources` -> `Add-ons`
 1. Search for `postgres` and attach a free "Hobby Dev" account
 
+<br/>
+![](docs/app_postgres.png)
```
Comment (froeba) [inline GETTING_STARTED.md:52]: These are not rendering correctly from what I can tell.  Could you remove the <br /> ? or add an additional empty line?

When viewing on your fork: https://github.com/cvasilak/workshop-web-app/blob/doc/GETTING_STARTED.md

Screenshot 2019-11-28 at 18 09 38

File: ARCHITECTURE.md (hunk 1 above)
Comment (cvasilak) [reply ARCHITECTURE.md]: it replaces the [static image](https://github.com/bertfroeba/workshop-web-app/blob/master/docs/app.png) with the animated gif.

**Comment by cvasilak** [comment]
both img rendering and animation gif should be displayed correctly now after latest commits.

**Review by froeba — APPROVED** [review]

**Pull request merged**